go.mod

```diff
@@ -1,78 +1,87 @@
 module github.com/redhatinsights/ros-ocp-backend
 
-go 1.20
+go 1.22.0
```

Comment:
@upadhyeammit I think it is better to avoid the minor version here, since we don't seem to have that granularity in the Dockerfile tag, thoughts?

Reply:
yeah.. actually its kind of like if we run the go mod tidy then it adds the minor version, updated it